### ISSUE TITLE
Error when starting background service on Android 14

### ISSUE THREAD

**Issue by ggbarcelos** [dotnet/maui#21215]
### Description

I am trying to start a background service on Android 14 using the following code:
**if (Build.VERSION.SdkInt >= BuildVersionCodes.Q)
{
    StartForeground(Constants.SERVICE_RUNNING_NOTIFICATION_ID, RegisterForegroundService(GetString(Resource.String.foregroundSemChamado), true), ForegroundService.TypeMediaPlayback);
}
else
{
    StartForeground(Constants.SERVICE_RUNNING_NOTIFICATION_ID, RegisterForegroundService(GetString(Resource.String.foregroundSemChamado), true));
}
**
However, I am receiving the following error:
**Java.Lang.SecurityException: 'Starting FGS with type mediaPlayback callerApp=ProcessRecord{ccb3c8f 10781:br.trueit.pythagoras/u0a409} targetSDK=34 requires permissions: all of the permissions allOf=true [android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK] '
**
I made the correct declarations both in the AndroidManifest and by explicitly asking for permissions at runtime. Still, I am facing this problem. Any help to solve this would be greatly appreciated.
My AndroidManifest:
<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
<uses-permission android:name="android.permission.FOREGROUND_SERVICE.MEDIA_PLAYBACK" />
Inside <application>
<service
   android:name="SAPHMovelDispositivo.EquipeServico"
   android:enabled="true" 
   android:exported="false"
   android:foregroundServiceType="mediaPlayback" />

### Version with bug

8.0.7 SR2

### Is this a regression from previous behavior?

Yes, this used to work in .NET MAUI

### Last version that worked well

8.0.7 SR2

### Affected platforms

Android

### Affected platform versions

Android 14

**Comment by jfversluis** [comment]
I don't think this is a bug in .NET MAUI. If anything this would be a .NET for Android bug, but right now I don't have any reason to think this is a bug to begin with but something that is not setup correctly in your project. As such, converting this to a discussion.

I am using this similar setup for my [ScreenRecording plugin](https://github.com/jfversluis/Plugin.Maui.ScreenRecording/) maybe there is a hint in there on something that you are still missing.
